themes/_core/js/checkout-payment.js

```diff
@@ -112,6 +112,8 @@ class Payment {
   confirm() {
     var option = this.getSelectedOption();
     if (option) {
+      $('.custom-checkbox').click(false);
```

Comment:
Stopping propagation can crash some modules. a button just need to be disabled after the first click and restored in case of failure

Reply:
I need help on this please.

Reply:
I delete this line